README.md

```diff
@@ -0,0 +1,38 @@
+# Geohashi
+
+Go(lang) clone of [geohash-int](https://github.com/yinqiwen/geohash-int).
+
+## Precision
+
+This library allows you to select the precision of the hash you want to create, up to a maximum of 26 bits.
+
+The following table shows the uncertainty for the given bit-precision:
```

Comment:
You should explain that this is the maximum uncertainty, or extend the table to show uncertainty at equator + another column for e.g. New York